.github/workflows/test.yml

```diff
@@ -70,8 +70,8 @@ jobs:
         run: make
       - name: Install
         run: sudo make install
-      - name: Validate examples
-        run: limactl validate ./examples/*.yaml
+      - name: Validate examples (except vmnet.yaml)
+        run: find examples -name '*.yaml' | grep -v 'vmnet.yaml' | xargs limactl validate
```

Comment:
Why does `vmnet.yaml` need to be excluded? It seems to validate fine.

Reply:
Because it needs VDE socket to be present

Reply:
> Because it needs VDE socket to be present

Ah yes, I completely forgot that I have the socket locally when I tested this.